shared/src/main/kotlin/org/jetbrains/kotlin/konan/target/Configurables.kt

```diff
@@ -98,6 +98,7 @@ interface AppleConfigurables : Configurables, ClangFlags {
     val arch get() = targetString("arch")!!
     val osVersionMin get() = targetString("osVersionMin")!!
     val osVersionMinFlagLd get() = targetString("osVersionMinFlagLd")!!
+    val stripFlags get() = targetString("stripFlags")!!
```

Comment:
Generally everything is OK. The last two things:
1. Use `targetList` instead of `targetString`. Otherwise things might break in case of several flags. Note that `Command` construction should also be changed to something like `Command(strip, *stripFlags.toTypedArray(), executable)`.
2. Squash your commits, please :)

Reply:
Thanks a lot for the feedback ;) It was a bit in hurray. But you're absolute right about those two points.